docs/modules/ROOT/pages/how-tos/vsphere/resize-masters.adoc

```diff
@@ -0,0 +1,28 @@
+= Resize Master/Control Plane systems
+
+[abstract]
+This page describes how to rescale the Master/Control Plane systems of an existing OCP4 IPI on vSphere cluster.
+
+== Requirements
+
+[options="interactive"]
+* [ ] Access to vCenter
+* [ ] Access to the cluster
+
+== Procedure
+
+This procedure must be repeated for each master system.
+
+. Drain the node.
+. Update the corresponding `Machine` object to reflect the desired state. This WON'T actually change the VM, so it can be done while the node is draining.
+** The relevant parameters are found under `spec.providerSpec.value`.
+** For Memory, update `memoryMiB`.
+** For CPU, update `numCPUs`.
+. Once the node is drained, shut down the machine.
+** Either via `oc debug node/...`, then `chroot /host` and `poweroff`...
+** .. or via vCenter, select `Power`, `Shut down Guest OS`.
+. Change the desired VM parameters.
```

Comment:
Which parameters and where?

Reply:
> Is there an upstream documentation we can link to? Either on docs.openshift.com or in the RedHat knowledge base.

No, the upstream docs say "If you want to resize the masters, set up a new cluster" :clown_face: 

That it can be done manually we only got from Red Hat Support (which we can't link to)

Reply:
Ok, forget about the link. But can you clarify on the vm parameters that have to be changed?

Reply:
Reworded, see latest version